A87-grpc-metrics-xds-outlier-detection.md

```diff
@@ -0,0 +1,64 @@
+A87: gRPC OTel Metrics for xDS Outlier Detection
+---
+* Author(s): Pardhu Konakanchi, Sai Teja
+* Approver:
+* Status: Draft
+* Implemented in: Core, Go
+* Last updated: 03-04-25
```

Comment:
Also, please start a thread for this gRFC on the grpc-io mailing list, and then add a link to that thread here, as per the gRFC process.